merge_requests.go

```diff
@@ -111,6 +111,8 @@ type ListMergeRequestsOptions struct {
 	State   *string `url:"state,omitempty" json:"state,omitempty"`
 	OrderBy *string `url:"order_by,omitempty" json:"order_by,omitempty"`
 	Sort    *string `url:"sort,omitempty" json:"sort,omitempty"`
+	Search  *string `url:"search,omitempty" json:"search,omitempty"`
```

Comment:
I don't see `Search` as a valid option in the docs, are you sure it's an option here? Can you point me to the docs or code showing that?